README.md

```diff
@@ -1,2 +1,104 @@
-# rust-vmm-container
-Container with all dependencies required for running rust-vmm crates integration tests. 
+# rust-vmm-dev-container
+
+**rust-vmm-dev** is a container with all dependencies used for running
+rust-vmm integration tests.
+
+The container is available on Docker Hub and has support for x86_64 and
+aarch64 platforms.
+Pull the latest version by running:
+
+```bash
+docker pull fandree/rust-vmm-dev
```

Comment:
I just created a `rustvmm` organization on Docker hub. You are one of the owners.

Reply:
kewl, will use it.

Reply:
Added issue for that #2 as there are some more changes that are required for doing the switch.